lib/enhance-assert.js

```diff
@@ -27,6 +27,7 @@ const formatter = context => {
 		.reverse();
 };
 
+// Todo Should be removed
```

Comment:
Is this something to address in this PR?

Reply:
We can remove this function. I was not sure if it is being used anywhere else, where I haven't seen it.

Reply:
No this should be the only place.